zebrad/tests/acceptance.rs

```diff
@@ -940,9 +952,8 @@ fn full_sync_test(network: Network, timeout_argument_name: &'static str) -> Resu
 /// If `check_legacy_chain` is true,
 /// make sure the logs contain the legacy chain check.
 ///
-/// If `enable_mempool_at_height` is `Some(Height(_))`,
-/// configure `zebrad` to debug-enable the mempool at that height.
-/// Then check the logs for the mempool being enabled.
+/// Configure `zebrad` to debug-enable the mempool based on `mempool_behavior`,
+/// Then check the logs for the expected `mempool_behavior`.
```

Comment:
Optional: capitalization typo

```suggestion
/// then check the logs for the expected `mempool_behavior`.
```

Reply:
Fixed in 00c754237.